tests/_support/Language/ab-CD/Allin.php

```diff
@@ -10,8 +10,19 @@
  */
 
 return [
-    'one' => 'Pyramid of Giza',
-    'tre' => 'Colossus of Rhodes',
-    'fiv' => 'Temple of Artemis',
-    'sev' => 'Hanging Gardens of Babylon',
+    'one'  => 'Pyramid of Giza',
+    'tre'  => 'Colossus of Rhodes',
+    'fiv'  => 'Temple of Artemis',
+    'sev'  => 'Hanging Gardens of Babylon',
+    'file' => [
+        'with\.' => [
```

Comment:
Why does `'with\.'` need `\`?

In dot array syntax, keys can contain a dot without escaping.
https://codeigniter4.github.io/CodeIgniter4/helpers/array_helper.html#dot_array_search